python3_dev_features.rst

```diff
@@ -18,25 +18,33 @@ gnashing of teeth.
 
 To explain this, we need to review the different kinds of parameters that a
 python function can be defined to take. The simplest is a positional
-parameter. The following function has two positional poarameters::
+parameter. The following function has two positional poarameters
```

Comment:
can you fix the poarameters -> parameters typo? (it was there before, but since you are editing this anyway)